manifests/00-ingress-credentials-request.yaml

```diff
@@ -117,7 +125,11 @@ apiVersion: cloudcredential.openshift.io/v1
 kind: CredentialsRequest
 metadata:
   name: openshift-ingress-alibabacloud
-  namespace: openshift-ingress-operator
+  namespace: openshift-cloud-credential-operator
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
```

Comment:
I don't think these will do anything for Alibaba but I'd prefer if this wasn't added as I don't see why it would be ibm-cloud-managed.

Reply:
I've [dropped them][1].

[1]: https://github.com/openshift/cluster-ingress-operator/pull/692#issuecomment-999338209